templates/includes/sidebar.html

```diff
@@ -1,4 +1,15 @@
 <ul class="sidebar-nav">
+  {% if user.is_authenticated %}
+    <li>
+      User: {{ user.username }}
```

Comment:
Add a hyperlink here to satisfy the following requirement from the task description:
`If the driver clicks on username - the corresponding Driver detail page must open.`